### ISSUE TITLE
Enable to set access level when adding users in to group from .csv files 

### ISSUE THREAD

**Issue by tomlebl** [nomad-nmr/nomad-server#91]
Currently access level is set by default to user-b when when users are added to group via .csv file import. 
It would be better to add access level and other priorities through a dialog when .csv file is imported.

**Comment by hunxjunedo** [comment]
hey, I'm having trouble setting up the dev environment, docker compose is tuck at `RUN chown -R node.node /app/node_modules `. Would be nice if you could help me with that.

**Comment by tomlebl** [comment]
There was a reason why I added this RUN command into front-end dockerfile while ago but it seems that it became obsolete in meantime. The app starts on my system even if I comment it out.  I push it to the main branch. Pull and try again. Let me know how it goes.
This is valuable feedback. Cheers

**Comment by hunxjunedo** [comment]
alright so I've successfully been able to set it up properly on my system, now I have a question regarding the issue: how do we take input of the access level ? should it be one single input for all or one by one each ?

**Comment by tomlebl** [comment]
I am glad to hear that you have managed to set dev environment on your computer.
I assume that add users button in group table would open a modal where you can select the .csv file with users and drop down menu (antd/Select) that allows to choose access level (user, user-b, admin-b or admin). The selected access level would be assigned to all users on the import list. At the moment, all users get user-b by default. 
I hope that makes sense. If not than  let me know. 
This slightly bigger issue it might be good to put it on a new branch.

**Comment by hunxjunedo** [comment]
I think the API test is failing as now the endpoint expects different a different body format.

**Comment by tomlebl** [comment]
@hunxjunedo It looks like. One test is failing. I will have a look properly tomorrow and will let you know.
I appreciate your help.

**Comment by hunxjunedo** [comment]
> @hunxjunedo It looks like. One test is failing. I will have a look properly tomorrow and will let you know.
> I appreciate your help. 

Alright, by the way I think this test is supposed to fail as the api changed.

**Comment by tomlebl** [comment]
That's right but we need to amend the test that it works with current version of API. I will do that once I check your work.

**Comment by tomlebl** [comment]
@hunxjunedo I have checked your code and looks good but when I was reviewing it, I have noticed that we likely try to solve non-existing issue because  what the user requested is already possible by other means. I sent him an e-mail to find out if the issue is still there or not. Until I hear from him I will leave this open but it might be that I will just close it without merging your code. I am sorry. However, I would be more than happy to assign you another one if you want to.

**Comment by hunxjunedo** [comment]
@tomlebl got it, now I'm looking for another issue. Let me know if an issue is a priority, I'll try to work on that.
